src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -41,19 +55,25 @@
   private static final String START_ATTRIBUTE = "start";
   private static final String DURATION_ATTRIBUTE = "dur";
   private static final String TEXT_TAG = "text";
-  // TODO: Use actual video id.
-  private static final String TEST_VIDEO_ID = "3ymwOvzhwHs";
+  private static final String PARAM_LECTURE = "Lecture";
```

Comment:
it seems like our params have generally been lowercase with dashes, probably good idea to be consistent

Reply:
I think this is supposed to be a `ENITITY_KIND`

Reply:
Gotcha. Lowercased. I'm not sure about naming it ENTITY_KIND though since I'm creating transcript entities. I think Nathan has a similar format with the params constant in his Comment class.